src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -19,20 +19,25 @@
 import seedu.address.model.person.Phone;
 import seedu.address.model.tag.Tag;
 import seedu.address.model.transaction.Amount;
+import seedu.address.model.util.Date;
 
 public class ParserUtilTest {
     private static final String INVALID_NAME = "R@chel";
     private static final String INVALID_PHONE = "+651234";
     private static final String INVALID_EMAIL = "example.com";
     private static final String INVALID_AMOUNT = "91a";
     private static final String INVALID_TAG = "#friend";
+    private static final String INVALID_DATE_1 = "11 May 2019";
+    private static final String INVALID_DATE_2 = "11.11.2020";
+    private static final String INVALID_DATE_3 = "30/02/2020";
 
     private static final String VALID_NAME = "Rachel Walker";
     private static final String VALID_PHONE = "123456";
     private static final String VALID_EMAIL = "rachel@example.com";
     private static final String VALID_AMOUNT = "5";
     private static final String VALID_TAG_1 = "friend";
     private static final String VALID_TAG_2 = "neighbour";
+    private static final String VALID_DATE = "02/02/2020";
```

Comment:
Must it be 02/02/2020? Is it possible to make dates without leading zeros (e.g. 2/2/2020) acceptable as well?